vxyowsup/whatsapp.py

```diff
@@ -147,6 +151,21 @@ def onMessage(self, messageProtocolEntity):
 
     @ProtocolEntityCallback("receipt")
     def onReceipt(self, entity):
+        '''receives confirmation of delivery to human'''
+        # shady
+        # getting too many receipts
```

Comment:
Is this still an issue? I'm wondering whether we actually need to be calling `.toLower()`? It seems like we should be sending acks when we receive messages, not when we receive receipts?